jsbridge/src/main/kotlin/de/prosiebensat1digital/oasisjsbridge/extensions/SetTimeoutExtension.kt

```diff
@@ -45,6 +52,13 @@ internal class SetTimeoutExtension(private val jsBridge: JsBridge) {
 
         // Same as for setTimeout but with setTimeoutHelper repeat parameter set to true
         JsValue.newFunction(jsBridge, "cb", "msecs", """
+            // undefined, null and wrong variable type (e.g. string) are valid values for timeout == no delay
+            // "1000" string is converted into a number
+            const TIMEOUT_MAX = 2 ** 31 - 1;
+            msecs *= 1; // Coalesce to number or NaN
+            if (!(msecs >= 1 && msecs <= TIMEOUT_MAX)) {
```

Comment:
Is it the same as `isNaN(msecs)`?

Reply:
You mean the `if` statement? Not really, it checks for positive and less than TIMEOUT_MAX. NaN would only check for NaN, so negatives would slip through. I borrowed the code snippet from node.js setTimeout implementation :)